project-chip/connectedhomeip
# [Telink]: Migrate to upstream zephyr (docker part)

#### Problem
Used to Telink builds use custom fork of Zephyr: https://github.com/rikorsev/zephyr/tree/telink_b91_tlsr9518adk80d_internal
Now, Telink platform support is available on upstream Zephyr: https://github.com/zephyrproject-rtos/zephyr
So, Telink Matter solution migrates to upstream Zephyr.

#### Change overview
This particular PR contains changes related to docker images:
chip-build-telink
chip-build-vscode

#### Testing
Manually. Steps:
1) Build chip-build-telink image
2) Run chip-build-telink 
3) Check if telink lighting example builds successfully
4) Flash Telink tlsr9518adk80d board with built binary
5) Check if it works fine with chip-tool
